File: c/agree.c
// Agree.c
// Program to take a char as input and tell agreed or not based on that input.

#include <stdio.h>

int main(void)
{
    char c;
    // Take input chars
    printf("Do you agree? (y/n) ");
    scanf("%c", &c);

    // Check if agreed
    if (c == 'y' || c == 'Y')
    {
        printf("Agreed!\n");
    }
    else if (c == 'n' || c == 'N')
    {
        printf("Not Agreed!\n");
    }

    return 0;
}

File: python/agree.py
# Lecture 06 - "Python"
# Program 06

s = input("Do you agree? ").lower()

if s in ["y", "yes"]:
    print("Agreed")
elif s in ["n", "no"]:
    print("Not agreed")
else:
    print("Please choose only (yes/no).")